modules/apps/object/object-test/src/testFunctional/tests/ObjectCustomLayouts.testcase

```diff
@@ -21,6 +21,82 @@ definition {
 		}
 	}
 
+	@description = "LPS-168228 - Verify it's possible to add many custom fields in the custom layout"
+	@priority = "4"
+	test CanAddManyFieldsOnLayout {
+		task ("Given an object with 20 fields is created") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 168228",
+				objectName = "CustomObject168228",
+				pluralLabelName = "Custom Objects 168228");
+
+			for (var n : list "1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16,17,18,19,20") {
+				ObjectAdmin.addObjectFieldViaAPI(
+					fieldBusinessType = "Text",
+					fieldLabelName = "Custom Field${n}",
+					fieldName = "customField${n}",
+					fieldType = "String",
+					isRequired = "false",
+					objectName = "CustomObject168228");
+			}
+		}
+
+		task ("And given an layout is created in the object") {
+			ObjectAdmin.addObjectLayoutViaAPI(
+				layoutName = "Layout Name",
+				objectName = "CustomObject168228");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 168228");
+
+			ObjectAdmin.goToLayoutsTab();
+
+			ObjectAdmin.selectKebabMenuOption(kebabOption = "View");
+
+			ObjectAdmin.markLayoutAsDefault();
+		}
+
+		task ("And given all the custom fields are added in the layout") {
```

Comment:
You can merge these two given tasks. Something like "And given a layout is created with all custom fields added" Since both tasks are dealing with the same layout, it makes sense to have this portion in one task.